.github/workflows/deploy.yml

```diff
@@ -0,0 +1,71 @@
+# storoku:ignore
+
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [main]
+  workflow_run:
+    workflows: [Releaser]
+    types: [completed]
+    branches: [main]
+  workflow_dispatch:
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  # apply staging on pushes to main, plan otherwise
+  # staging:
+  #   uses: ./.github/workflows/terraform.yml
+  #   with:
+  #     env: staging
+  #     workspace: staging
+  #     did: did:web:staging.signer.storacha.network
+  #     apply: ${{ github.event_name != 'pull_request' }}
+  #   secrets:
+  #     aws-account-id: ${{ secrets.STAGING_AWS_ACCOUNT_ID }}
+  #     aws-region: ${{ secrets.STAGING_AWS_REGION }}
+  #     region: ${{ secrets.STAGING_AWS_REGION }}
+  #     private-key: ${{ secrets.STAGING_PRIVATE_KEY }}
+  #     signing-key: ${{ secrets.STAGING_SIGNING_KEY }}
+  #     cloudflare-zone-id: ${{ secrets.STAGING_CLOUDFLARE_ZONE_ID }}
+  #     cloudflare-api-token: ${{ secrets.STAGING_CLOUDFLARE_API_TOKEN }}
+
+  warm-staging:
+    uses: ./.github/workflows/terraform.yml
+    with:
+      env: warm-staging
+      workspace: warm-staging
+      network: warm
+      did: did:web:staging.signer.warm.storacha.network
+      apply: ${{ github.event_name != 'pull_request' }}
+    secrets:
+      aws-account-id: ${{ secrets.WARM_STAGING_AWS_ACCOUNT_ID }}
+      aws-region: ${{ secrets.WARM_STAGING_AWS_REGION }}
+      region: ${{ secrets.WARM_STAGING_AWS_REGION }}
+      private-key: ${{ secrets.WARM_STAGING_PRIVATE_KEY }}
+      signing-key: ${{ secrets.WARM_STAGING_SIGNING_KEY }}
```

Comment:
two things:
1. We should not be storing the private key of the wallet in github. Ideally, we reference a secret name here instead. 
2. I can't tell what these keys represent, one is the service identity (we derive a did from it) and the other is the wallet private key. Lets land on names that are easier to reason about while also addressing point 1.

Reply:
> 1. We should not store the private key of the wallet in github. Can need to reference a secret name here instead? This wallet will contain real money, and we need to limit the places we keep its key.

Good point. Right now storoku both creates the secret in AWS secrets manager and configures the service to get access to the secret at runtime. We could add a new flag to storoku to disable the provisioning part while keeping the service configuration functionality. I created https://github.com/storacha/storoku/issues/29 to track this.